xml/System.Runtime.CompilerServices/InternalsVisibleToAttribute.xml

```diff
@@ -46,8 +46,9 @@
     <remarks>
       <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- Ordinarily, types and members with `internal` or `private protected` scope (in C#) and `Friend` and `Private Protected` scope (in Visual Basic) are visible only in the assembly in which they are defined. The <xref:System.Runtime.CompilerServices.InternalsVisibleToAttribute> attribute makes them also visible to the types in a specified assembly, which is known as a friend assembly. This only applies to `internal` (`Friend` in VB) or `private protected`(`Private Protected` in VB) methods only, but not `private` ones.
```

Comment:
@gewarren I believe `private protected` is  correct.

Reply:
From [this test](https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/Compilers/CSharp/Test/Emit/PrivateProtected.cs#L83) in Roslyn, it implies that `private protected` is indeed accessible via IVTs. I haven't tried that though.